go-controller/pkg/ovn/master_test.go

```diff
@@ -612,7 +612,7 @@ subnet=%s
 			clusterController.SCTPLoadBalancerUUID = sctpLBUUID
 			clusterController.SCTPSupport = true
 
-			_ = clusterController.joinSubnetAllocator.AddNetworkRange(ovntest.MustParseIPNet("100.64.0.0/16"), 3)
+			_ = clusterController.joinSubnetAllocator.AddNetworkRange(ovntest.MustParseIPNet("100.64.0.0/16"), 29)
```

Comment:
This should be like above (32-3) so we know where the value came from, and ideally use a constant from previous comment. There are multiple instances of this, just tagging the first one.

Reply:
personally I think the 32-3 should just be removed and just use the result. There's a comment above the code that says its reserving 3 bits.